cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/LedgerEvents/Gov/PredefinedAbstainDRep.hs

```diff
@@ -145,7 +169,7 @@ delegateToAlwaysAbstain execConfig epochStateView configurationFile socketPath s
   let voteDelegationCertificatePath = baseDir </> "delegation-certificate.delegcert"
   void $ H.execCli' execConfig
     [ "conway", "stake-address", "vote-delegation-certificate"
-    , "--always-abstain"
+    , flag
```

Comment:
Why `flag` is a `String`? What other values do you expect here? It would be better to use a type better representing possible values of the flag.

Reply:
Good point, there are very few options after all. Fixed here: https://github.com/IntersectMBO/cardano-node/pull/5809/commits/2a618fd9fc234025365ec970cc7b90d669f70846